src/scripts/shell/dimAppCtrl.controller.js

```diff
@@ -132,19 +132,12 @@ function DimApp(ngDialog, $rootScope, loadingTracker, dimPlatformService, $inter
 
       var language = translate ? vm.settings.language + '/' : ''; // set language
 
-      var file;
-      try {
-        file = require('app/views/' + language + name + '.template.html');
-      } catch (e) {
-        file = require('app/views/en/' + name + '.template.html');
-      }
-
       if (result) {
         result.close();
       } else {
         ngDialog.closeAll();
         result = ngDialog.open({
-          template: file,
+          template: require('app/views/' + language + name + '.template.html'),
```

Comment:
Yeah, it does mean that every possible file will be included, but because these files are loaded externally anyway, 'included' means very little for the final JS bundle.